etc/release.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# usage: ./etc/release.sh [new version string]
+
+# exit if any command fails
+set -e
+
+# ensure we are on master before releasing
+git checkout master
+
+# regenerate documentation with new version string
+./etc/generate-docs.sh ${1}
```

Comment:
can we add a check to ensure a version string is passed in?

Reply:
added 👍